apps/api/src/app/workflows-v2/usecases/validate-content/prepare-and-validate-content/prepare-and-validate-content.usecase.ts

```diff
@@ -198,7 +199,7 @@ export class PrepareAndValidateContentUsecase {
       result[item] = [
         {
           issueType: StepContentIssueEnum.MISSING_VALUE,
-          message: `[${item}] No Value was submitted to a required control.`,
+          message: `${capitalize(item)} is missing.`,
```

Comment:
Yay or nay for the full stop? I couldn't decide which is cleaner. Deferring to a full stop for now. 🤔

Reply:
Designs show without the full stop. I've removed.